misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/ParserTestFormatter.java

```diff
@@ -33,6 +37,63 @@ public void test(String fileName, String path) throws IOException {
         super.testParserResources(path);
     }
 
+//    // Uncomment to run a subset of test cases.
```

Comment:
is this intentionally commented?

Reply:
Yes, this is to make the testing of subsets easier.